### ISSUE TITLE
Upgrade to play 2.5

### ISSUE THREAD

**Issue by hanswesterbeek** [delving/narthex#96]
The way we deal with webjars (see #95) is starting to hurt, and fixing it depends on getting this upgrade done.

**Comment by hanswesterbeek** [comment]
Last to-do is to fix the sending of messages through the websocket back to the client. The current implementation relied on knowledge that is private to play-akka, it just sent messages to a channel with a name (`/system/websockets/*`) that happened to be picked up by play internals. It was never documented to work:

```scala
def broadcastRaw(message: String) = context.system.actorSelection("/system/websockets/*") ! message
```

So when one upgrades, that breaks.

The solution seems to be that, rather than broadcasting, we should send the message to ActorRef of the `DataSocketActor`

**Comment by hanswesterbeek** [comment]
Also, upgrade the dependency on play-webjars

**Comment by hanswesterbeek** [comment]
Fixed by commit a6adc0dc8bb73a95090fc
